honeybee_energy/properties/face.py

```diff
@@ -0,0 +1,116 @@
+# coding=utf-8
+"""Face Energy Properties."""
+from ..construction import OpaqueConstruction
+from ..lib.default.room import generic_costruction_set
+
+
+class FaceEnergyProperties(object):
+    """Energy Properties for Honeybee Face.
+
+    Properties:
+        construction
+        materials
+        r_value
```

Comment:
It looks like you could have take advantage of a subclass not to rewrite construction, materials, r_value and u_value separately for every object type.